_posts/2022-12-31-end-of-year-update.markdown

```diff
@@ -49,26 +49,26 @@ When Qt6 came around, they switched to CMake, which is a lot nicer for cross-com
+**Upstream port patches.** We made a good initial effort this year to start upstreaming patches, and we managed to [land our targets](https://lists.gnu.org/archive/html/config-patches/2022-09/msg00001.html) in the GNU config repo, used by autotools. In the coming year, we would like to start upstreaming our targets to the respective toolchain packages (binutils, GCC, LLVM, libtool and rustc), and if successful, start upstreaming patches to other packages.
 
-**Port a web browser and work towards self hosting.** While we made great improvements in this department, including the addition of the `links` browser mentioned in the August update, we aim to have one of the common browsers (Chrome, Firefox, WebKitGTK) running by the end of 2023. We also would like to get more common development programs in, like `git`. While we also had this goal at the end of 2019, it turned out that the given timeframe was unrealistic. However, due to the progress achieved in the past 3 years and the addition of someone working almost dedicated on porting software, we feel that we can get this goal done in 2023.
+**Port a web browser and work towards self hosting.** While we made great improvements in this area, including the addition of the `links` browser mentioned in the August update, we aim to have one of the common browsers (Chrome, Firefox, WebKitGTK) running by the end of 2023. We also would like to get more common development programs in, like `git`. While we also had this goal at the end of 2019, it turned out that the given timeframe too ambitious. However, due to the progress achieved in the past three years, like the porting of the X stack and GTK 3, and the addition of someone to the team who likes to work on porting software, we feel that we can get this goal done in 2023.
 
-**Add more drivers.** Specifically graphics and networking related. Together with a few [LAI](https://github.com/managarm/lai) quirks and USB related quirks, this is the biggest issue for running on real hardware, a lack of drivers. Recently, work has been put into writing more network drivers, and some members are looking into adding at least modesetting drivers for Intel iGPUs. This is certainly an area we'd like help in, even if it is just testing on real hardware for quirks.
+**Add more drivers.** Specifically graphics and networking related. Together with a few [LAI](https://github.com/managarm/lai) quirks and USB related quirks, a lack of drivers is the biggest issue for running on real hardware. Recently, work has been put into writing more network drivers, and some contributors are looking into adding at least modesetting drivers for Intel iGPUs. This is certainly an area we would benefit from additional contributions, even if it is just testing on real hardware for quirks.
```

Comment:
don't repeat the word 'quirks'

Reply:
'at least modesetting' just say 'modesetting'

Reply:
'an area we' -> 'an area where we'

Reply:
'even if it is just testing on real hardware for quirks' -> 'even just testing on real hardware'

Reply:
Applied, thanks!